app-src/index.js

```diff
@@ -0,0 +1,28 @@
+var initPlayerRouter = require('dce-player-router');
+var pathExists = require('object-path-exists');
+
+var seekMethodPath = ['player', 'videoContainer', 'seekToTime'];
+
+var router = initPlayerRouter({
+  seeking: {
+    seekParamName: 't',
+    seekResponder: seekWhenPaellaIsReady
+  }
```

Comment:
It would helpful to include a comment in the header of this javascript about the purpose of this javascript, to make it clear for future people & help desk looking through this JS. For  "example: http://server/player/watch.html?id=74b6c02f-afbb-42bc-8145-344153a1792e#t=10-20, Where t specifies where to start and stop. 10 meaning "start at ten secondsand20` meaning "stop at 20 seconds" in this example.